1.8/administration/virtual-networks/ip-per-container.md

```diff
@@ -0,0 +1,194 @@
+---
+layout: page
+nav_title: IP-per-Container
+post_title: Configuring IP-per-Container in Virtual Networks
+menu_order: 10
+---
+
+In Enterprise DC/OS, the virtual network feature is enabled by default. The default configuration provides two overlay networks `dcos-1` and `dcos-2`, whose YAML configuration is as follows:
+
+```yaml
+  dcos_overlay_network:
+    vtep_subnet: 192.15.0.0/20
+    vtep_mac_oui: 70:B3:D5:00:00:00
+    overlays: 
+      - name: dcos-1
+        subnet: 192.168.0.0/17
+        prefix: 24
+      - name: dcos-2
+        subnet: 192.168.128.0/17
+        prefix: 24
+```
+
+**Note:** To use virtual networks in DC/OS you MUST make sure to use a recent Linux kernel (3.9 or above) as well as Docker version 1.11 on the agent nodes.
+
+Each overlay network is identified by a canonical `name`. Containers launched on an overlay network will get an IP address from the `subnet` allocated to the overlay network. To remove the dependency on a global IPAM (IP address management system), the overlay `subnet` is further split into smaller subnets, with each subnet being allocated to an agent. The agents can then use a host-local IPAM to allocate IP addresses from their respective subnets to containers launched on the agent and attached to the given overlay. The `prefix` determines the size of the subnet (carved the overlay `subnet`) allocated to each
+agent.
+
+In the configuration above, each virtual network is allocated a /17 subnetwork (in the “subnet” field), which is then divided in /24 subnetworks to be used in each host that will be part of the network (in the “prefix” field). This allocates 8 bits (32 total minus 24 allocated for “prefix”) to designate the endpoint (container) inside a host for a maximum of 255 endpoints per host. It also allocates 7 bits (24 minus 17) for hosts that will be members of this overlay network, which provides a maximum of 127 hosts. These values can be configured to adapt to each installation’s needs.
+
+# Adding overlay networks during installation
+
+Currently (as of version 1.8), DC/OS virtual networks can only be added and configured at install time. You can override the default network or add additional virtual networks by modifying your `config.yaml` file:
```

Comment:
We don't support upgrades currently since we write these networks in the overlay replicated log. I think I was trying to bring this out clearly in the `re-install` section.